release_notes.py

```diff
@@ -43,6 +43,9 @@
 
 
 def add_dot(text):
+    if not text:
```

Comment:
What’s that for ?

Reply:
incase the text is empty, the function code crashes (saw that in another PR).
should never happen, since we have validations for that (in the validator step).